catalog/ui/catalog-ui-search/pom.xml

```diff
@@ -348,7 +353,9 @@
                             !org.abego.treelayout.*,
                             *
                         </Import-Package>
-                        <Export-Package/>
+                        <Export-Package>
+                            org.codice.ddf.catalog.ui.metacard
```

Comment:
❗️ This package contains concrete classes with interfaces.  Move classes to an `impl` sub package to avoid them being exported.  I would also suggest making this an internal package for now.  So `org.codice.ddf.catalog.ui.metacard.internal` and `org.codice.ddf.catalog.ui.metacard.impl` for example.

Reply:
I'll reorganize the classes and interfaces.